Doc/whatsnew/3.11.rst

```diff
@@ -99,6 +99,20 @@ Optimizations
   almost eliminated when no exception is raised.
   (Contributed by Mark Shannon in :issue:`40222`.)
 
+* Method calls with keywords are now faster due to bytecode
+  changes which avoid creating bound method instances. Previously, this
+  optimization was applied only to method calls with purely positional
+  arguments.
+  (Contributed by Ken Jin and Mark Shannon in :issue:`26110`, based on ideas
+  implemented in PyPy.)
+
+CPython bytecode changes
+========================
+
+* Added a new :opcode:`CALL_METHOD_KW` opcode.  Calls a method in a similar
+fashion as :opcode:`CALL_METHOD`, but also supports keyword arguments.  Works
+in tandem with :opcode:`LOAD_METHOD`.
+
```

Comment:
Checkout the sphinx warning:

https://github.com/python/cpython/pull/26159/checks?check_run_id=2602412992

Reply:
Thank you. Fixed.